x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -249,4 +223,38 @@ describe('CaseActionBar', () => {
     userEvent.click(screen.getByTestId('property-actions-ellipses'));
     expect(queryByText('Delete case')).toBeInTheDocument();
   });
+
+  it('shows the external incident action', async () => {
+    const connector = caseConnectors['servicenow-1'];
+    const { push, ...connectorWithoutPush } = connector;
+
+    const props = {
+      ...defaultProps,
+      caseData: { ...defaultProps.caseData, connector: connectorWithoutPush },
+    };
+
+    render(
+      <TestProviders>
+        <CaseActionBar {...props} />
+      </TestProviders>
+    );
+
+    userEvent.click(screen.getByTestId('property-actions-ellipses'));
+
+    await waitFor(() => {
+      expect(screen.getByTestId('property-actions-popout')).toBeInTheDocument();
+    });
+  });
+
+  it('does not show the external incident action', async () => {
```

Comment:
The connector of the `caseData` is the `none` connector. `useGetCaseConnectors` response does not contain the `none` connector so it will not shown the external service action.